.github/workflows/issue_pm_ai.yml

```diff
@@ -0,0 +1,114 @@
+name: Issue PM AI
+
+on:
+  issues:
+    types: [opened]
+
+permissions:
+  contents: read
+  issues: write
+
+jobs:
+  respond:
+    # Only run for issues opened in the main repository (not forks)
+    if: github.repository == 'internetarchive/openlibrary'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      
+      - name: Load PM AI Instructions
+        id: pm
+        run: |
+          {
+            echo "INSTRUCTIONS<<EOF"
+            cat .github/prompts/issue_pm_instructions.md
+            echo "EOF"
+          } >> "$GITHUB_OUTPUT"
+
+      - name: Generate issue response (GitHub Models)
+        id: generate
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Is this needed?  I believe that `GITHUB_TOKEN` is always available to workflows by default.